lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -930,6 +934,33 @@ fromBlockfrostPP network BF.ProtocolParams{..} = do
                     }
         , .. }
 
+-- | Selects a minimum UTxO function that is appropriate for the current era.
+--
+-- TODO: [ADP-1994]
+--
+-- This function is currently hard-wired to select the Alonzo era minimum UTxO
+-- function, which computes a result based on the 'coinsPerUTxOWord' protocol
+-- parameter.
+--
+-- However, the Babbage era will switch to a minimum UTxO function that depends
+-- on the 'coinsPerUTxOByte' protocol parameter.
+--
+-- We should revise this function so that it's capable of selecting a minimum
+-- UTxO function that's appropriate for the current era.
+--
+getMinimumUTxOFunction
+    :: BF.ProtocolParams
+    -> Either BlockfrostError MinimumUTxO
+getMinimumUTxOFunction BF.ProtocolParams {_protocolParamsCoinsPerUtxoWord} =
+    minimumUTxOForAlonzoEra . Ledger.Coin
+        <$> intCast @_ @Integer _protocolParamsCoinsPerUtxoWord
+        <?#> "CoinsPerUtxoWord"
+  where
+    minimumUTxOForAlonzoEra :: Ledger.Coin -> MinimumUTxO
+    minimumUTxOForAlonzoEra coinsPerUTxOWord = minimumUTxOForShelleyBasedEra
+        ShelleyBasedEraAlonzo
+        def {Alonzo._coinsPerUTxOWord = coinsPerUTxOWord}
```

Comment:
Using `def { }` in production seems a bit sketchy, but if it's just for the time being then sure.